types/testing.go

```diff
@@ -171,19 +172,25 @@ func NewSignedMessageForTestGetter(ms MockSigner) func() *SignedMessage {
 // SomeCid generates a Cid for use in tests where you want a Cid but don't care
 // what it is.
 func SomeCid() cid.Cid {
-	b := &Block{}
-	return b.Cid()
+	obj, err := cbor.WrapObject([]int{42}, DefaultHashFunction, -1)
```

Comment:
this is always going to return the same value, no?  Why not just find out what it is and return that as a constant?  Then you don't have to have this panic code which will almost certainly never get called. Or see my other comment.

Reply:
Agreed, will add to a types-testing tech debt follow on issue.

Reply:
#3193